llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -580,6 +583,72 @@ static unsigned getMaxBitmapSize(const CounterMappingContext &Ctx,
   return MaxBitmapID + (SizeInBits / CHAR_BIT);
 }
 
+struct DecisionRow {
```

Comment:
add doxygen comment

Reply:
I have added comments in `DecisionRow`. Please give me suggestions.

Reply:
What does it represent in the MC/DC algorithm?

Reply:
Now it is `MCDCDecisionRecorder::DecisionRecord` (and made private)